azure/functions_worker/bindings/generic.py

```diff
@@ -0,0 +1,46 @@
+import typing
+
+from . import datumdef
+
+
+class GenericBinding:
+
+    @classmethod
+    def has_trigger_support(cls) -> bool:
+        return False
+
+    @classmethod
+    def check_input_type_annotation(cls, pytype: type) -> bool:
+        return issubclass(pytype, (str, bytes))
+
+    @classmethod
+    def check_output_type_annotation(cls, pytype: type) -> bool:
+        return issubclass(pytype, (str, bytes, bytearray))
+
+    @classmethod
+    def encode(cls, obj: typing.Any, *,
+               expected_type: typing.Optional[type]) -> datumdef.Datum:
+        if isinstance(obj, str):
+            return datumdef.Datum(type='string', value=obj)
+
+        elif isinstance(obj, (bytes, bytearray)):
+            return datumdef.Datum(type='bytes', value=bytes(obj))
+
+        else:
+            raise NotImplementedError
+
+    @classmethod
+    def decode(cls, data: datumdef.Datum, *, trigger_metadata) -> typing.Any:
+        data_type = data.type
+
+        if data_type == 'string':
```

Comment:
if `data_type` is `undefined` it is string

Reply:
This is different from RPC `dataType`.  `Datum.type` is always defined.

Reply:
Is `dataType. Datum.type` set to  `string` default?

Reply:
Worker ignores `dataType` completely and only pays attention to whats in `TypedData`.